tools/verify_assembler_traces.py

```diff
@@ -272,7 +272,7 @@ def ConvertMovRdImm(matches):
       # Fall back to a LLVM making the right decision.
       return "mov {}, #{}".format(*matches)
     llvm_mc_instruction_converters[:0] = [
-        # The ARM ARM specifies that if <Rn> is PC in either an ADD or SUB
+        # The ARM specifies that if <Rn> is PC in either an ADD or SUB
```

Comment:
??

Reply:
This is a common acronym. The Arm ARM is the Arm Architecture Reference Manual. Basically, what defines an Arm. Latest version is https://developer.arm.com/documentation/ddi0487/ha/?lang=en

Reply:
Interesting... https://www.google.com/search?q=%22arm+arm+specifies%22

Fwiw, if you had written `Arm ARM` (or `ARM-ARM`), my tool wouldn't have tripped on it. (It's also possible to just tell my tool that `ARM ARM` is ok.)

(Today I actually ran into `Jar Jar` for the first time..., but I knew that one...)

Anyway, dropped.